src/checks_windows.go

```diff
@@ -174,6 +186,11 @@ func PasswordChanged(user, date string) (bool, error) {
 	return !res, err
 }
 
+func WindowsFeature(feature string) (bool, error) {
+	res, err := command("Get-WindowsOptionalFeature -FeatureName " + feature + " -Online | Select-Object State | Select-String Enabled")
```

Comment:
Might be more reliable with `commandOutput`, like `return commandOutput("(Get-WindowsOptionalFeature -FeatureName " + feature + " -Online).State", "Enabled"`)